src/components/Nav.tsx

```diff
@@ -64,7 +66,41 @@ export const MobileNav: React.FC = () => {
         </div>
       </header>
 
-      <MobileSidebar isOpen={isNavOpen} />
+      <div
+        css={[isNavOpen ? tw`block` : tw`hidden`, tw`w-full`, tw`md:hidden`]}
+      >
+        <>
+          {sidebarContent.map((section, i) => (
+            <React.Fragment key={i}>
+              {section.title != null && (
+                <h5 tw="px-4 my-2 text-foreground text-sm font-bold">
+                  {section.title}
+                </h5>
+              )}
+
+              <ul tw="mb-8">
+                {section.pages.map(page => (
+                  <li key={page.slug} onClick={() => setIsNavOpen(!isNavOpen)}>
+                    <Link
+                      href={page.slug}
+                      css={[
+                        tw`text-gray-700 text-sm`,
+                        tw`block px-4 py-2`,
+                        tw`hover:bg-gray-100 hover:text-foreground`,
+                        tw`focus:outline-none focus:bg-pink-100`,
+                        pathname === page.slug &&
+                          tw`bg-pink-100 text-pink-900 hover:bg-pink-100 border-r-2 border-pink-500`,
+                      ]}
+                    >
+                      {page.title}
+                    </Link>
+                  </li>
+                ))}
+              </ul>
+            </React.Fragment>
+          ))}
+        </>
+      </div>
```

Comment:
I'm not sure why the sidebar code is copy and pasted here? I think instead we should be listening for changes on the route and can close the sidebar if it changes.

This change also has the sidebar open when you first visit the page on mobile which isn't ideal.

Reply:
I didn't know I can listen to the page change event. I knew my solution had a smell, but wasn't clever enough to pull that up. I will update.